platform/product/product.go

```diff
@@ -60,14 +60,15 @@ var (
 		QDJVMC: "IIC",
 		// QDAND: // don't use it right now
 		// QDANDC: // don't use it right now
-		QDPHP: "PS",
-		QDJS:  "WS",
-		QDNET: "RD",
-		QDPY:  "PCP",
-		QDPYC: "PCC",
-		QDGO:  "GO",
-		QDRST: "RR",
-		QDCPP: "CL",
+		QDPHP:  "PS",
```

Comment:
I would recommend to refactor "isPaid" "supportsNative", "language name" and "image" from separate objects to a fields of some struct lets say "QodanaProduct", and just have a list of them. Currently to add a product too many separate fields need to be changed. For newcomers to qodana cli it's especially confusing, out of box one doesnt expect that there are N fields which need to be edited to make the system valid